src/components/OverallMaturity.tsx

```diff
@@ -47,7 +60,17 @@ export const OverallMaturity = () => {
             />
           </Grid>
           <Grid item xs={12}>
-            <ExportEntitiesForm />
+            <InfoCard title={<div style={{marginBottom: "-18px"}}>{header}</div>}>
```

Comment:
nit: move the styled div into `header` so this lines becomes simple:

```suggestion
            <InfoCard title={header}>
```

Reply:
right lol